### PR TITLE
[TFLite] Added coverage test for 16x8 quantizaion post-training mode.

### PR DESCRIPTION
**Pull request by wwwind**
Added a coverage test for 16x8 post-training quantization mode.

### CODE REVIEW

**Comment by akarmi** [comment]
@jdduke, please check this change. We discussed it as one of the requirements for the stable release of 16x8-bit quantization.

File: tensorflow/lite/testing/model_coverage/model_coverage_lib.py [hunk 1]
```diff
@@ -83,7 +83,8 @@ def _convert(converter, **kwargs):
   Args:
     converter: TFLiteConverter object.
     **kwargs: Additional arguments to be passed into the converter. Supported
-      flags are {"target_ops", "post_training_quantize", "quantize_to_float16"}.
+      flags are {"target_ops", "post_training_quantize",
+       "quantize_to_float16", "quant_16x8", "model_input_size"}.
```
Comment (jdduke) [inline tensorflow/lite/testing/model_coverage/model_coverage_lib.py:87]: Nit: Can we at least call this "quantize_16x8" or "post_training_quantize_16x8" for consistency with other "quantize" flags?

**Review by jdduke — CHANGES_REQUESTED** [review]
Looks good just one minor point on naming.

**Comment by wwwind** [comment]
Hi @jdduke Thanks for the review. I renamed. Please take a look.

**Review by jdduke — APPROVED** [review]

**Comment by gbaned** [comment]
@wwwind Can you please address Ubuntu Sanity errors? Thanks!

**Comment by wwwind** [comment]
Hi @gbaned 
This failure does not look relevant to my changes.

The test //tensorflow/core/grappler/costs:op_level_cost_estimator_test is failing, but this failure is due to timeout. 
`tensorflow/core/grappler/costs/op_level_cost_estimator_test.cc:984
Expected equality of these values:
  cost.compute_time
    Which is: 5000ns
  Costs::Duration(expected_compute_time)
    Which is: 4300ns
Softmax`

I checked that is passes locally.
Could you please re-run Ubuntu Sanity Checks on this PR ?

File: tensorflow/lite/testing/model_coverage/model_coverage_lib.py [hunk 2]
```diff
@@ -83,7 +83,8 @@ def _convert(converter, **kwargs):
   Args:
     converter: TFLiteConverter object.
     **kwargs: Additional arguments to be passed into the converter. Supported
-      flags are {"target_ops", "post_training_quantize", "quantize_to_float16"}.
+      flags are {"target_ops", "post_training_quantize",
+       "quantize_to_float16", "post_training_quantize_16x8", "model_input_size"}.
```
Comment (jdduke) [inline tensorflow/lite/testing/model_coverage/model_coverage_lib.py:87]: This is longer than 80 lines, can you fix?

**Review by jdduke — CHANGES_REQUESTED** [review]

**Comment by wwwind** [comment]
@jdduke fixed. Could you please re-approve ? Thanks!

**Review by jdduke — APPROVED** [review]
